gallery_dl/extractor/kemonoparty.py

```diff
@@ -450,7 +450,7 @@ class KemonopartyFavoriteExtractor(KemonopartyExtractor):
     def __init__(self, match):
         KemonopartyExtractor.__init__(self, match)
         self.favorites = text.parse_query(
-            match.group(2)).get("type") or self.config("favorites", "artist")
+            match.group(2)).get("type", self.config("favorites", "artist"))
```

Comment:
In a sense this is worse than before. because it *always* calls `self.config()` even if the URL query parameters have a `type`.

By using `a or b`, `b` only gets evaluated when `a` is empty and not at all when `a` has a "true" value.